4337/contracts/test/SafeMock.sol

```diff
@@ -123,21 +122,17 @@ contract Safe4337Mock is SafeMock, IAccount {
         bytes32,
         uint256 missingAccountFunds
     ) external onlySupportedEntryPoint returns (uint256) {
-        address entryPoint = msg.sender;
-
-        _validateReplayProtection(userOp);
-
```

Comment:
We don’t do replay protection checks in the module, so I removed this to compare apples to apples.